pytorch_lightning/tuner/batch_size_scaling.py

```diff
@@ -39,6 +39,8 @@ def scale_batch_size(
     batch_arg_name: str = "batch_size",
 ) -> Optional[int]:
     """See :meth:`~pytorch_lightning.tuner.tuning.Tuner.scale_batch_size`"""
+    raise MisconfigurationException("this is gone")
```

Comment:
Reminder to remove this.

Reply:
ah! must have come back while rebasing. removed!!